tests/unit/json_test.c

```diff
@@ -1443,6 +1443,32 @@ static void test_parse_object_double_and_trailing_comma(void)
     }
 }
 
+#define assert_json_strings(unescaped, escaped)        \
```

Comment:
Would be nice to add some verb to the name of the macro. Or at least something like `_eq` to the end.

Reply:
Alright, added.